carsurvey2/R/get_2colour_gradients.R

```diff
@@ -0,0 +1,80 @@
+#' Create 2 colour gradient palette
+#'
+#' Creates a list of  colour names (RGB). Generates a palate using two colour gradients and an optional grey neutral colour.
+#' Each half of the palette contains different shades of the colour submitted to the function. These are useful where two colour
+#' scales in the same palette are needed, e.g. likert type plots. By default, the main colours are shades of blue and yellow
+#' used in ONS outputs.
+#'
+#' @param n the number of colours needed
+#' @param colour1 the first colour in the scale - a numeric vector representing red, green and blue (max 255)
+#' @param colour2 the last colour in the scale - numeric vector representing red, green and blue (max 255).
+#' @param mid the middle value of the scale. If neutral_mid = TRUE this value will be represented in grey. Otherwise, this will be
+#' the endpoint of the first colour gradient (e.g. if n=6 and mid=3, there will be three shades of each colour)
+#' @param neutral_mid whether the middle of the scale should be represented in grey (logical, TRUE by default)
+#'
+#' @return RGB colours
+#'
+#' @export
+
+
+get_2colour_gradients <- function(n, colour1 = c(32, 96, 149), colour2 = c(226, 188, 34), mid, neutral_mid = TRUE) {
+  
+  if (!is.numeric(n) | length(n) > 1) {
+    stop("n is not a numeric value")
+  } else if (n < 2) {
```

Comment:
From a developer point of view, it might make sense to flip this to match the error?
```suggestion
  } else if (!n >= 2) {
```